deploy/cloudformation/elastic-agent-ec2-cspm-organization.yml

```diff
@@ -84,6 +84,13 @@ Resources:
           PolicyDocument:
             Version: "2012-10-17"
             Statement:
+              - Effect: Allow
+                Action:
+                  - iam:ListAccountAliases
+                  - iam:ListGroup
+                  - iam:ListRoles
+                  - iam:ListUsers
+                Resource: '*'
```

Comment:
There is a significant consideration when updating ClodFormation files:

For existing installations, the Elastic Agent will be updated but the CloudFormation will not. Without a workaround or fallback path, existing multi-account aws installations that are updated from 8.12 to 8.13 will stop working.